cmd/podman/validate/latest.go

```diff
@@ -7,9 +7,8 @@ import (
 
 func AddLatestFlag(cmd *cobra.Command, b *bool) {
 	// Initialization flag verification
-	cmd.Flags().BoolVarP(b, "latest", "l", false,
-		"Act on the latest container podman is aware of\nNot supported with the \"--remote\" flag")
-	if registry.IsRemote() {
-		_ = cmd.Flags().MarkHidden("latest")
+	if !registry.IsRemote() {
+		cmd.Flags().BoolVarP(b, "latest", "l", false,
+			"Act on the latest container podman is aware of\nNot supported with the \"--remote\" flag")
```

Comment:
I had troubles brain parsing the error message.
Could we reduce it to "--lastet is not supported for remote calls" or something similar?